regnety/dataset/imagenet.py

```diff
@@ -18,54 +20,77 @@
+    a callable object. That callable must take exactly two arguments: `image` and `target`
+    and must return two values corresponding to the same. 
+
+    If `augment_fn` argument is 'val', then the images woll be center cropped to 224x224.
 
     Args:
         tfrecs_filepath: list of filepaths of all TFRecords files
         batch_size: batch_size for the Dataset
         image_size: final image size of the images in the dataset
         augment_fn: function to apply to dataset after loading raw TFrecords
         num_classes: number of classes
-        randaugment: True if RandAugment is to be applied after other 
-            preprocessing functions. It is applied even if this is True and 
-            augment_fn is not 'default'. It is not applied in any case if this
-            argument is False.
+        
     """
     def __init__(
         self,
         tfrecs_filepath: List[str]  = None,
-        batch_size: int = 128,
-        image_size: int = 224,
+        batch_size: int = 1024,
+        image_size: int = 512,
         augment_fn: Union[str, Callable]  = "default",
-        num_classes: int = 10,
-        randaugment: bool = True,
+        num_classes: int = 1000,
     ):
 
         if (tfrecs_filepath is None) or  (tfrecs_filepath == []):
             raise ValueError("List of TFrecords paths cannot be None or empty")
+
         self.tfrecs_filepath = tfrecs_filepath
         self.batch_size = batch_size
         self.image_size = image_size
         self.augment_fn = augment_fn
         self.num_classes = num_classes
-        self.randaugment = randaugment
-        if self.randaugment:
-            self._augmenter = RandAugment(magnitude=5, num_layers=2)
+        
+        if self.augment_fn == 'default':
+            self.default_augment = True
+            self.val_augment = False
+            self.strength = 5
+        elif self.augment_fn == 'val':
+            self.default_augment = False
+            self.val_augment = True
+            self.strength = -1
+        else:
+            self.default_augment = False
+            self.val_augment = False
+            self.strength = -1
+        
+        
+        
 
-    def decode_example(self, example: tf.Tensor) -> dict:
-        """Decodes an example to its individual attributes
+    
+    def decode_example(self, example_: tf.Tensor) -> dict:
+        """Decodes an example to its individual attributes.
 
         Args:
             example: A TFRecord dataset example.
 
         Returns:
             Dict containing attributes from a single example. Follows
-            the same names as TFRECORDS_FORMAT.
+            the same names as _TFRECS_FORMAT.
         """
-        image = tf.cast(tf.io.decode_jpeg(example["image"]), tf.float32)
+
+        example =  tf.io.parse_example(example_, _TFRECS_FORMAT)
+        image = tf.reshape(tf.io.decode_jpeg(
+            example["image"]), (self.image_size, self.image_size, 3))
```

Comment:
Just want to ensure we are consistent on the use of single-quotes and double-quotes. If not we'd want to ensure it.

Reply:
Corrected wherever required